frontend/src/Components/VotingStateDisplay.tsx

```diff
@@ -32,7 +32,7 @@ const getRank = (voted: boolean, observer: boolean) => {
   return 1;
 };
 
-const getClassName = (voted: boolean, observer: boolean) => {
+const getClass = (voted: boolean, observer: boolean) => {
```

Comment:
i would stick here with either getClassName, getClassAttrib or getHtmlClass otherwise it feels to missleading/generic in the js/ts context

Reply:
Done